src/frontends/paddle/tests/test_models/gen_scripts/generate_linspace.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# linspace paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle
+import random
+import sys
+
+data_type = "float32"
+
+
+def linspace(name: str, start, stop, num, type=None):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        if isinstance(start, np.ndarray):
+            in_start = paddle.static.data(name="start", shape=[1], dtype=type)
+            in_stop = paddle.static.data(name="stop", shape=[1], dtype=type)
+            in_num = paddle.static.data(name="num", shape=[1], dtype='int32')
+        else:
+            in_start = paddle.full(name="start", shape=[1], fill_value=start, dtype=type)
+            in_stop = paddle.full(name="stop", shape=[1], fill_value=stop, dtype=type)
+            in_num = paddle.full(name="num", shape=[1], fill_value=num, dtype='int32')
+
+        out = paddle.linspace(in_start, in_stop, in_num)
+
+        cpu = paddle.static.cpu_places(1)
+        exe = paddle.static.Executor(cpu[0])
+
+        exe.run(paddle.static.default_startup_program())
+
+        outs = exe.run(feed={"Start": in_start, "Stop": in_stop, "Num": in_num}, fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=["Start", "Stop", "Num"], fetchlist=[out], inputs=[in_start, in_stop, in_num],
+                  outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def main():
+    start = random.randint(0, 10)
+    stop = random.randint(0, 10)
+    num = random.randint(1, 10)
+    linspace("linspace_1", start, stop, num, "int32")
+
+    start = random.uniform(0, 10)
+    stop = random.uniform(0, 10)
+    num = random.randint(1, 10)
+    linspace("linspace_2", start, stop, num, "float32")
+
```

Comment:
start and stop could be tensor of shape rank 1. Is there any difference in the serialized protobuf model w.r.t. a scaler input? If it is, please add a test case when the input is a tensor.

Reply:
and do you cover/consider the cases when num is 1 or 0?

Reply:
test sample has been added, but num=0 is not permitted by paddle.

float64 is not supported yet, I am working on this error.
![image](https://user-images.githubusercontent.com/69072522/220813994-aa27cce3-d193-4937-aed6-c4ecca6ae383.png)
I am a bit confused because float64 would have been converted to float32 in the first part of my code.